spec/nikeplus/models/activities_spec.rb

```diff
@@ -1,11 +1,11 @@
 require 'spec_helper'
 
 describe NikePlus::Activities do
-  let(:token) { "foobar" }
+  let(:token) { "thetoken" }
 
   it '#fetch_data retrieve data activities' do
     VCR.use_cassette("all_activities") do
-      activities = NikePlus::Activities.new().fetch({access_token: token, count: 999})
+      activities = NikePlus::Activities.new().fetch({access_token: token, count: 10000})
```

Comment:
Line is too long. [88/80]<br>Redundant curly braces around a hash parameter.<br>Do not use parentheses for method calls with no arguments.<br>Space inside { missing.<br>Space inside } missing.